iac-extensions/arm/src/test/resources/checks/TlsVersionCheck/Microsoft.DBfor_SQL_servers/test.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "type": "Microsoft.DBforMySQL/servers",
+      "apiVersion": "2017-12-01",
+      "name": "[variables('serverName')]",
```

Comment:
Please change the name to describe the reason behind raising an issue. Same for the other resources.